cli/src/commands/extensions/mod.rs

```diff
@@ -178,26 +178,29 @@ fn ask_permissions(extension: &Extension) -> Result<()> {
 
     println!("The `{}` extension requires the following permissions:", extension.name());
 
-    fn print_permissions_list<S: Display>(key: &str, detail: &str, items: Option<&Vec<S>>) {
+    fn print_permissions_list<S: Display>(key: &str, resource: &str, items: Option<&Vec<S>>) {
```

Comment:
This isn't the only place where the `Permission::get` method is used though. I do not want to duplicate this logic, especially because it has some rather fragile edge cases?

So I don't think we'd gain anything here?